core/vibes/soul/sections/cart/client.tsx

```diff
@@ -558,66 +567,95 @@ function CounterForm({
+                  lineItem.quantity === 1
+                    ? 'opacity-50'
+                    : 'hover:bg-[var(--cart-counter-background-hover,hsl(var(--contrast-100)/50%))]',
+                )}
+                disabled={lineItem.quantity === 1}
+                name="intent"
+                type="submit"
+                value="decrement"
+              >
+                <Minus
+                  className={clsx(
+                    'text-[var(--cart-counter-icon,hsl(var(--contrast-300)))] transition-colors duration-300',
+                    lineItem.quantity !== 1 &&
+                      'group-hover:text-[var(--cart-counter-icon-hover,hsl(var(--foreground)))]',
+                  )}
+                  size={18}
+                  strokeWidth={1.5}
+                />
+              </button>
+              <span className="flex w-8 select-none justify-center focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-[var(--cart-focus,hsl(var(--primary)))]">
+                {lineItem.quantity}
+              </span>
+              <button
+                aria-label={incrementLabel}
+                className={clsx(
+                  'group rounded-r-lg bg-[var(--cart-counter-background,hsl(var(--background)))] p-3 transition-colors duration-300 hover:bg-[var(--cart-counter-background-hover,hsl(var(--contrast-100)/50%))] focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-[var(--cart-focus,hsl(var(--primary)))] disabled:cursor-not-allowed',
+                )}
+                name="intent"
+                type="submit"
+                value="increment"
+              >
+                <Plus
+                  className={clsx(
+                    'text-[var(--cart-counter-icon,hsl(var(--contrast-300)))] transition-colors duration-300',
+                  )}
+                  size={18}
+                  strokeWidth={1.5}
+                />
+              </button>
+            </div>
+            <button
+              aria-label={deleteLabel}
+              className="group -ml-1 mt-1.5 flex h-8 w-8 shrink-0 items-center justify-center self-start rounded-full transition-colors duration-300 hover:bg-[var(--cart-button-background,hsl(var(--contrast-100)))] focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-[var(--cart-focus,hsl(var(--primary)))] focus-visible:ring-offset-4"
+              name="intent"
+              type="submit"
+              value="delete"
+            >
+              <Trash2
+                className="text-[var(--cart-icon,hsl(var(--contrast-300)))] group-hover:text-[var(--cart-icon-hover,hsl(var(--foreground)))]"
+                size={20}
+                strokeWidth={1}
+              />
+            </button>
+          </div>
+          {!!lineItem.inventoryMessages?.outOfStockMessage && (
```

Comment:
Instead of coercing to a boolean with `!!` can we do `!= null`? That would be more consistent with the rest of the codebase.